airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/commands.py

```diff
@@ -105,7 +105,7 @@ def validate_environment(is_local: bool, use_remote_secrets: bool):
                 raise click.UsageError(f"When running in a CI context a {required_env_var} environment variable must be set.")
     if use_remote_secrets and os.getenv("GCP_GSM_CREDENTIALS") is None:
         raise click.UsageError(
-            "You have to set the GCP_GSM_CREDENTIALS if you want to download secrets from GSM. Set the --use-remote-secrets option to false otherwise."
+            "You have to set the GCP_GSM_CREDENTIALS if you want to download secrets from GSM. See README for instructions ('Setting up connector secrets access'). Set the --use-remote-secrets option to false otherwise."
```

Comment:
📚 We could go as far as providing a link directly. But this seems more than enough! 

Good call updating the error message

Reply:
I figured a link in a CLI which won't turn it into an actual link (will stay full url) is kind of annoying. But I could be wrong!

Reply:
I guess it depends on your terminal, but in iterm2 you can cmd+click urls which I do all the time so it would be a nice touch. However, I would have searched for `Setting up connector secrets access` in the repo based on this message and it would have turned up the right thing as well.